### ISSUE TITLE
Problems plotting MagIC 3.0 pole plots with multiple rows in the location table with the same name

### ISSUE THREAD

**Issue by njarboe** [PmagPy/PmagPy#393]
Here is an example of a contribution that is typical for a pole only contribution. It has two locations but four rows in the location table. The "Guyana Permo-Triassic Dykes" location is listed only once and should have its pole plotted by make_magic_plots.

The next three rows in the locations table are for the "La Quinta Formation Lavas". The first row, with all the sites averaged, and then the normal and reversed poles. I imagine we could use all of the three as plots, but the most important one is the first one that includes both reversed and normal.

Then it would also be nice to have a pole plot that has two points, the "Guyana Permo-Triassic Dykes" location and the main "La Quinta Formation Lavas".

So using the naming convention following the current one, we would want:
'LO:_' + 'Guyana Permo-Triassic Dykes' + '_POLE_map.' + fmt - The one pole given in the locations table 
'LO:_' + 'La Quinta Formation Lavas' + '_POLE_map.' + fmt - The combined pole. The second row in the table. Can be distinguished from the other ones with the same location name by the fact that the dir_polarity column is empty.

'MC:_' + '12012' + '_POLE_map.' + fmt - The two poles above plotted on the same plot. MC stands for magic contribution.

Then for the ones that have values in the dir_polarity column, One could create those and name them 
'LO:_' + 'La Quinta Formation Lavas' +'_n' +'_POLE_map.' + fmt
'LO:_' + 'La Quinta Formation Lavas' +'_r' +'_POLE_map.' + fmt

depending on the value in the dir_polarity column.

Magic Contribution file:
[magic_contribution_12012.txt](https://github.com/PmagPy/PmagPy/files/2199642/magic_contribution_12012.txt)

locations.txt file:
[locations.txt](https://github.com/PmagPy/PmagPy/files/2199641/locations.txt)

Currently on the pmag server, make_magic_plots uses polemap_magic.py to make this plot from this contribution:
![lo___ty__pole_map](https://user-images.githubusercontent.com/1278789/42788756-3d277ccc-8916-11e8-9ec2-4b92a52da94c.png)
Which looks empty and has an empty file name.

**Comment by ltauxe** [comment]
do you have a clever way to automatically search for these cases?  are they properly distinguished by method codes?  
lisa

**Comment by njarboe** [comment]
They are not currently labeled with method codes. There is a DE-POLE method code that could/should be added to rows that describe poles, but I don't see that being used. I could add DE-POLE-R with a description of "The pole calculated using only the reverse directions/VGPs of the same named location dataset" and DE-POLE-N with "The pole calculated using only the normal directions/VGPs of the same named location dataset". We could encourage/require people to used these in the future when reporting poles and could be added to past datasets in the future. When online editing is available doing this would probably take a few days and could be accurately done by an undergrad student.

**Comment by njarboe** [comment]
Thanks Lori

**Comment by moonshoes87** [comment]
The reason that no poles show up in the plot is because only rows with 'result_type' of 'a' are kept.  The result types in this contribution are all individual.  If I remove that restriction, I get this plot (and we can adjust the title to say something useful):

![mc _12012_ty _pole_map](https://user-images.githubusercontent.com/1836578/42914771-ff5a4fd6-8ab0-11e8-966a-94a262c87c47.png)

@ltauxe, how do you want to handle the average vs. individual results in these cases?  Maybe we could plot individual results if there is no sites table?

@njarboe, I will work on getting those other plots as well.

**Comment by moonshoes87** [comment]
Ok, I'm pushing a fix that ignores the result_type but only if isServer.  However, we might want to use a more sophisticated way of plotting only appropriate results.  @ltauxe, feel free to give your input on this.

When run locally, everything should still work exactly as it did before.

@njarboe, I did the plot filenames slightly differently than your spec -- mainly just adding \_TY\_ for consistency.  I also added titles to the plots, which can also be altered as needed.

    1  saved in  MC:_12012_TY:_POLE_map.png
    2  saved in  LO:_Guyana Permo-Triassic Dykes_TY:_POLE_map_.png
    3  saved in  LO:_La Quinta Formation Lavas_TY:_POLE_map_.png
    4  saved in  LO:_La Quinta Formation Lavas_n_TY:_POLE_map_.png
    5  saved in  LO:_La Quinta Formation Lavas_r_TY:_POLE_map_.png
